bin/amphtml-update.py

```diff
@@ -421,15 +421,6 @@ def ParseRules(repo_directory, out_dir):
 			for _list in field_val:
 				descendant_lists[_list.name] = []
 				for val in _list.tag:
-
-					# Skip tags specific to transformed AMP.
-					if 'I-AMPHTML-SIZER' == val:
-						continue
-
-					# The img tag is currently exclusively to transformed AMP, except as descendant of amp-story-player.
-					if 'IMG' == val and 'amp-story-player-allowed-descendants' != _list.name:
-						continue
```

Comment:
Considered below two points:
- native `img` is supported by AMPHTML now.
- `i-amphtml-sizer` can be present in SSR-transformed HTML.